storage/local/persistence_test.go

```
@@ -135,7 +135,7 @@ func testPersistLoadDropChunks(t *testing.T, encoding chunk.Encoding) {
 			}
 		}
 		// Load all chunk descs.
-		actualChunkDescs, err := p.loadChunkDescs(fp, 0)
+		actualChunkDescs, _ := p.loadChunkDescs(fp, 0)
```

Let's not ignore the error.